js/main.js

```diff
@@ -51,55 +52,57 @@ const _hour = _minute * 60;
 
 // setInterval allows you to do certain action every period of time (like cronjob)
-setInterval(function() {
-
-  // Check is current time + 10 minutes smaller than the starting time
-  if(now+1000*60*10 < start) {
-    // If it is, show the box and start counting down
-    $('.countdown').show(400);
-  } else {
-    liveActive = true;
-    // Otherwise, show the livestream and abjust the height of it to be in
-    // 19:6 aspect ratio
-    // When done hiding the countdown show the livestream, make it take 400ms so there is a smooth transition
-    if(liveActive) {
-      showLive();
+setInterval(
+  () => {
+    // Check is current time + 10 minutes smaller than the starting time
+    if (now + 1000 * 60 * 10 < start) {
+      // If it is, show the box and start counting down
+      $('.countdown').show(400);
+    } else {
+      liveActive = true;
+      // Otherwise, show the livestream and abjust the height of it to be in
+      // 19:6 aspect ratio
+      // When done hiding the countdown show the livestream, make it take 400ms so there is a smooth transition
+      if (liveActive) {
+        showLive();
+      }
     }
-  }
-  now += 1000;
+    now += 1000;
 
-  const dist = start - now;
-  const days = Math.floor(dist / _day);
-  const hours = Math.floor( (dist % _day) / _hour );
-  const mins = Math.floor( (dist % _hour) / _minute );
-  const secs = Math.floor( (dist % _minute) / _second );
+    const dist = start - now;
+    const days = Math.floor(dist / _day);
+    const hours = Math.floor(dist % _day / _hour);
+    const mins = Math.floor(dist % _hour / _minute);
+    const secs = Math.floor(dist % _minute / _second);
 
-  let str  = 	(days > 0) 		? '<span>' + days + ' Days</span>' 		: '';
-  str +=		(hours > 0) 	? '<span>' + hours + ' Hours</span>'	: '';
-  str +=		(mins > 0) 		? '<span>' + mins + ' Minutes</span>'	: '';
-  str +=		'<span>' + secs + ' Seconds</span>';
+    let str = days > 0 ? `<span>${days} Days</span>` : '';
+    str += hours > 0 ? `<span>${hours} Hours</span>` : '';
+    str += mins > 0 ? `<span>${mins} Minutes</span>` : '';
+    str += `<span>${secs} Seconds</span>`;
 
-  $('.countdown .countdown__counter').html(str);
-}, 1000); // Set the time of the interval, in ms, so 1000 (1s)
+    $('.countdown .countdown__counter').html(str);
+  },
+  1000
+); // Set the time of the interval, in ms, so 1000 (1s)
 
 // Show the livefeed
-function showLive() {
-  if(!liveShown){
-    $('.countdown').hide(400, function() {
-      $('.livestream').show(0, function() {
+function showLive () {
+  if (!liveShown) {
+    $('.countdown').hide(400, () => {
+      $('.livestream').show(0, () => {
         $('video').height(0);
-        $('video').animate({height: $('video').width() * (9 / 16), display: 'block'}, 1000);
+        $('video').animate({ height: $('video').width() * (9 / 16), display: 'block' }, 1000);
       });
     });
     liveShown = true;
   }
 }
 
-function getHash() {
+function getHash () {
```

Comment:
Is there a convention where you separate the fn name from param's like this?
Never seen this style before